Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php

```diff
@@ -68,9 +68,16 @@ public function __construct(
          * @return PropertyCollectionInterface Property values, indexed by their name
          */
         public readonly PropertyCollectionInterface $properties,
+        /**
```

Comment:
```suggestion

        /**
```

Reply:
ah, sorry, I did't really check the surrounding code, just stumbled upon the missing empty line.

+1 for moving to `@param` annotations, otherwise the IDE won't show these